src/groups/mwc/mwcc/mwcc_orderedhashmap.t.cpp

```diff
@@ -1190,7 +1191,7 @@ static void testN3_profile_GoogleBenchmark(benchmark::State& state)
     // This case can be used to profile the component.
 
     typedef mwcc::OrderedHashMap<int, int> MyMapType;
-    MyMapType                              map(state.range(0), s_allocator_p);
+    MyMapType map(static_cast<int>(state.range(0)), s_allocator_p);
```

Comment:
Perhaps similar here?  I'm honestly surprised that line 1198 doesn't trigger a signed/unsigned comparison warning though.

This one is probably okay on its own.  Generally an explicit cast means "Trust me, this conversion is safe and I mean it," which is why they have such long and scary names in C++, compared with C-style casts.  That's all well and good, and these casts are indeed safe, you're right!  But I'm not convinced we mean them---I think many of these `int`s are incidental, and really what we meant was `whatever type state.range(0) returns`.  So whenever we can avoid casts (without breaking our public interface, without changing our behavior, and without making very large changes to the codebase), I tend to think it's better to do so.